src/main/java/com/smartparkinglot/backendservice/services/driverservice/DriverService.java

```diff
@@ -0,0 +1,15 @@
+package com.smartparkinglot.backendservice.services.driverservice;
+
+import com.smartparkinglot.backendservice.domain.Driver;
+import org.springframework.http.ResponseEntity;
+
+import javax.management.InstanceAlreadyExistsException;
+import java.util.List;
+
+public interface DriverService {
+    ResponseEntity<List<Driver>> getAllDrivers();
```

Comment:
Like i said before comment service should not be responsible to return a response. I am adding this comment for clarity what i mean.
```java
List<Driver> getAllDrivers();
```